### DISCUSSION TITLE
Post your screenshots here

### DISCUSSION THREAD

**Discussion by rnkn** [rnkn/olivetti#68] [General]
<img width="868" alt="olivetti screenshot" src="https://user-images.githubusercontent.com/1256849/78529941-0e2ad580-7826-11ea-9034-c3f33788d62c.png">

<img width="1282" alt="olivetti screenshot" src="https://user-images.githubusercontent.com/1256849/78750288-490e4400-79b3-11ea-93fb-efb01bfac769.png">

**Comment by rnkn** [comment]
A simple screenshot

<img width="1282" alt="simple-screenshot" src="https://user-images.githubusercontent.com/1256849/84850582-6dfdf200-b09b-11ea-83d4-5d9cf92455eb.png">

**Comment by overvale** [comment]
Off-topic, but how did you setup your mode-line to look like that?

**Comment by rnkn** [comment]
@olivertaylor a couple of minor things:

Add `(internal-border-width . 20)` to `default-frame-alist` to get the frame padding.

Customize the `mode-line` face to have an overline.

Then the `mode-line-format` is:
```elisp
(:eval
 (mode-line-align
  (format-mode-line
   (list " " mode-line-buffer-identification " " mode-line-modified " " mode-name))
  (format-mode-line
   (list minor-mode-alist " " mode-line-misc-info))))
```
Where `mode-line-align` is defined as:
```elisp
(defun mode-line-align (left right)
  "Return a string with LEFT and RIGHT at the edges of the
current window."
  (format (format "%%s %%%ds" (- (window-total-width) (length left) 2))
          left right))
```

I really appreciate your work on Textplay!

**Comment by overvale** [comment]
Thanks! I've just recently starting getting in to Emacs (a little) so I've been playing around with your package a bit. Nice work.

**Comment by rnkn** [comment]
@olivertaylor cool, if you're using Fountain Mode I'm interested in hearing experiences from new Emacs users. I try to make it as user-friendly as possible.

**Comment by overvale** [comment]
@rnkn Would be glad to write-down my impressions as I go. What's the best way to get them to you? Here? As a "general feedback" issue in `fountain-mode`? I'm "in between" writing projects so I haven't had much of a chance to put it through its paces, just kicking the tires at the moment.

**Comment by rnkn** [comment]
@olivertaylor if you run into specific bugs, bug reports on rnkn/fountain-mode would be good, otherwise I don't mind. Email is fine too.

**Comment by Yevgnen** [comment]
<img width="1386" alt="Screen Shot 2022-01-16 at 1 43 36 PM" src="https://user-images.githubusercontent.com/16749790/149648786-a04a4db0-5802-4ac9-a6c9-9b608176811c.png">

**Comment by madand** [comment]
![emacs-olivetti-org](https://user-images.githubusercontent.com/2280844/150846600-3921cb6f-3012-4c19-93d3-21bac8e5975b.png)
Literate Emacs config in Org Mode with Olivetti. The theme is [modus-operandi](https://protesilaos.com/emacs/modus-themes) (built-in in Emacs 28) with slightly adjusted background colors. Also, `olivetti-style` is set to `fancy` and `olivetti-fringe` face's background color is changed.
